assets/webpack.config.ts

```diff
@@ -1,127 +1,128 @@
+const path = require("path");
 
 const relativePath = path.resolve.bind(path, __dirname);
 const config: Configuration = {};
-const isProduction = process.env.NODE_ENV === 'production';
+const isProduction = process.env.NODE_ENV === "production";
 
 module.exports = config;
 
 config.entry = {
-  game: './src/app/Game.ts',
-  test: './src/app/Test.ts',
-  index: './src/index.ts',
-  vendor: ['phoenix', 'phoenix_html'],
+  game: "./src/app/Game.ts",
+  test: "./src/app/Test.ts",
+  index: "./src/index.ts",
+  vendor: ["phoenix", "phoenix_html"]
 };
 
 config.output = {
-  path: relativePath('../priv/static'),
-  filename: 'js/[name].js',
+  path: relativePath("../priv/static"),
+  filename: "js/[name].js"
 };
 
 config.plugins = [
   new CheckerPlugin(),
   new webpack.HashedModuleIdsPlugin(),
   new webpack.optimize.CommonsChunkPlugin({
-    name: 'vendor',
+    name: "vendor"
   }),
   new webpack.optimize.CommonsChunkPlugin({
-    name: 'runtime',
+    name: "runtime"
   }),
   new CopyWebpackPlugin([
     {
-      context: 'static',
-      from: '**/*',
-    },
-  ]),
+      context: "static",
+      from: "**/*"
+    }
+  ])
 ];
 
 if (!isProduction) {
   config.plugins.push(
     new BundleAnalyzerPlugin({
-      openAnalyzer: false,
+      openAnalyzer: false
     })
   );
 }
 
 const tsRule = {
   test: /\.tsx?$/,
-  loader: 'awesome-typescript-loader',
+  loader: "awesome-typescript-loader",
   options: {
     useBabel: true,
     useCache: true,
-    reportFiles: ['src/**/*.{ts,tsx}'],
-  },
+    reportFiles: ["src/**/*.{ts,tsx}"]
+  }
 };
 
 const jsRule = {
   test: /\.jsx?$/,
   exclude: /(node_modules|bower_components)/,
   use: {
-    loader: 'babel-loader',
+    loader: "babel-loader",
     options: {
       plugins: [
         [
-          '@babel/transform-async-to-generator',
+          "@babel/transform-async-to-generator",
           {
-            module: 'bluebird',
-            method: 'coroutine',
-          },
-        ],
-      ],
-    },
-  },
+            module: "bluebird",
+            method: "coroutine"
+          }
+        ]
+      ]
+    }
+  }
 };
 
 const cssRule = {
   test: /\.css$/,
   use: [
     {
-      loader: 'style-loader',
+      loader: "style-loader",
       options: {
-        sourceMap: true,
-      },
+        sourceMap: true
+      }
     },
     {
-      loader: 'css-loader',
+      loader: "css-loader",
       options: {
         sourceMap: true,
-        importLoaders: 1,
-      },
+        importLoaders: 1
+      }
     },
     {
-      loader: 'postcss-loader',
-    },
-  ],
+      loader: "postcss-loader"
+    }
+  ]
 };
 
 const elmRule = {
   test: /\.elm$/,
   exclude: [/elm-stuff/, /node_modules/],
   use: {
-    loader: 'elm-webpack-loader',
+    loader: "elm-webpack-loader",
     options: {
-      cwd: relativePath('elm'),
+      cwd: relativePath("elm"),
       debug: !isProduction,
-    },
-  },
+      pathToMake: relativePath("node_modules/.bin/elm-make")
```

Comment:
This is the only line that was actually change by me and is the reason for the PR.